src/HonzaBotner/appsettings.BotDev.json

```diff
@@ -13,7 +13,7 @@
         ]
     },
     "CVUT": {
-        "AppBaseUrl": "http://fit-cvut-discord-test.herokuapp.com"
+        "AppBaseUrl": "https://fit-cvut-discord-test.herokuapp.com"
```

Comment:
This change is more likely incorrect one. Our app is in heroku accepting and responding in http and some heroku higher level functionality manages https

Reply:
I think you are assigned a `*.herokuapp.com` wildcard certificate, as I am hosting FIS bot on heroku as well and https works fine

Reply:
But this may vary for your use case, it's just what's working for me

Reply:
If you say so I will believe you, To be honest, I dont want to touch it if it works :D

Reply:
SSL is imo nice thing to have, here I have a heroku app deployed with zero SSL related config: 
https://vse-verification.herokuapp.com

Reply:
Should I leave it on `http` or keep the updated version with `https`? I have it rewritten on heroku too without issues, but I have barely tested something on it.

Reply:
If it works, I would leave it. We can open another issue/pr where we will discus this thing. Generally it is not good idea to mix in one PR multiple things.